internal/email/generators.go

```diff
@@ -85,6 +90,31 @@ func generateRegistrationConfirmMessage(userName, websiteUrl, token string) herm
 	}
 }
 
+func generateNewEmailConfirmMessage(userName, websiteUrl, token string) hermes.Email {
```

Comment:
Please consider removing 'New' from the method name to align with other methods.